### DISCUSSION TITLE
[Kavita] When importing CBL store missing issue info to generate a missing items admin view on the list's page.

### DISCUSSION THREAD

**Discussion by Cducharme84** [Kareadita/Kavita#2962] [Ideas]
### Idea Description

With reading lists imported via CBL being a static and well defined set of what's needed for completion the CBL provides all that's needed and Kavita's already matching the list versus the libraries on the server for completion. So we'd just need for backend work to use a new store for the missing issues in imported reading lists, things like Volume name, Issue #, and order within the original CBL would be all that's needed from the original CBL to store to create the potential uses I'll describe below. We would also need to relate to the list's ID.

The current page for an individual reading list has all issues currently imported showing for all users, we could have the option enablable in the page's setting screen for admin users to be able to check and "fill in" the spots where those issues lay with placeholder "missing" image as the cover. This will allow admins to see at a glance where the gaps are and they would need to fill if they are interested.

This information would also be able to be used to update the server's reading list accordingly when the missing issues are scanned and then added to library allowing for an auto-update to match the CBL at import (or if on the JSON version of CBL in the future the current version from the repo when Kavita does its periodic check) without server admins/users with CBL permissions updating CBL manually.

I think this would work best as admin only truly as they are the only users able to potentially add the books to the library collections but there would maybe even be benefit for non-admin users to see if they can read those outside of their admin's server if they're truly passionate about the story.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

~1.25 years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by falo2k** [comment]
An additional side to this tracking would be monitoring when books are removed from a reading list - comics are deleted, or filenames change such that the same volume is recognised as a new volume, or the volume gets renamed by updated metadata (even though IDs remain constant).  Some form of administration screen much like we already have Media Issues would be appropriate.  It could track issues that have fallen out of a list or newly identified matches, and allow the user to apply fixes (remapping, or leaving as blank placeholders).
